server/views/cms/cms.py

```diff
@@ -0,0 +1,94 @@
+import logging
+from flask import jsonify, request
+import requests
+import html
+
+from server import app
+from server.util.request import api_error_handler
+
+logger = logging.getLogger(__name__)
+BASE_URL = app.config['PAYLOAD_API_URL']
+API_KEY = app.config['PAYLOAD_API_KEY']
+
+if not BASE_URL or not API_KEY:
+    error_message = 'Configuration error: PAYLOAD_API_URL and/or PAYLOAD_API_KEY are missing.'
+    logger.error(error_message)
```

Comment:
Logging the API key doesn't seem like a good idea, it just doesn't feel right. Instead, I'll make the error message more explicit by clearly stating what exactly is missing to avoid any ambiguity.